api/models/process.go

```diff
@@ -22,6 +23,7 @@ type Process struct {
 	Name    string    `json:"name"`
 	Ports   []string  `json:"ports"`
 	Release string    `json:"release"`
+	Size    int64     `json:"size"`
```

Comment:
I wonder if there is name that could more strongly hint this is the size of memory and not cpu shares or something else. `MemSize` ? `RequestedMemory` ? `ProvisionedMemory` ? Just some food for thought.